File: app/src/main/java/com/timotiusoktorio/pencake/data/model/Price.java
package com.timotiusoktorio.pencake.data.model;

@SuppressWarnings("unused")
public class Price {

    private String size;
    private Integer price;

    public Price() {
    }

    public Price(String size, Integer price) {
        this.size = size;
        this.price = price;
    }

    public String getSize() {
        return size;
    }

    public void setSize(String size) {
        this.size = size;
    }

    public Integer getPrice() {
        return price;
    }

    public void setPrice(Integer price) {
        this.price = price;
    }

    public String getPriceAndSize() {
        return "$ " + price + " (" + size + ")";
    }

    public String getSizeAndPrice() {
        return size + " - $ " + price;
    }
}

File: app/src/main/java/com/timotiusoktorio/pencake/data/model/Price.kt
package com.timotiusoktorio.pencake.data.model

data class Price(var size: String = "", var price: Int = 0) {

    val priceAndSize: String
        get() = "$ $price ($size)"

    val sizeAndPrice: String
        get() = "$size - $ $price"
}